test/with_dummyserver/test_no_ssl.py

```diff
@@ -21,13 +21,22 @@ def test_simple(self) -> None:
             assert r.status == 200, r.data
 
 
-class TestHTTPSWithoutSSL(HTTPSDummyServerTestCase, TestWithoutSSL):
+class TestHTTPSWithoutSSL(HTTPDummyProxyTestCase, TestWithoutSSL):
```

Comment:
You might want to add some assertions to existing HTTPS proxy tests as well, specifically it'd be good to assert that the `InsecureProxyWarning` isn't raised when using the proper configuration for secure proxies.

Reply:
Thanks, I have added them.